masterplan_tools/models/geojson.py

```diff
@@ -0,0 +1,127 @@
+"""
+Geojson response model and its inner parts are defined here.
+"""
+from typing import Any, Generic, Literal, TypeVar
+
+import geopandas as gpd
+from pydantic import BaseModel
+from shapely.geometry import mapping
+from shapely.geometry.base import BaseGeometry
+
+
+class Geometry(BaseModel):
+    """Geometry representation for GeoJSON model."""
+
+    type: Literal["Polygon", "MultiPolygon", "Point"]
+    coordinates: list[Any] = []
+
+    # @classmethod
+    # def from_dict(cls, dict : dict[str, any]) -> 'Geometry':
+    #   return cls(type=dict.type, coordinates=dict.coordinates)
```

Comment:
Methods which were not implemented after all should be removed instead of being commented.